userguide/advanceduse/dav.rst

```diff
@@ -10,26 +10,78 @@ At the moment, there is no carddav implementation directly accessible from the U
+ 
+4) Edit or create the following configuration variables.
+    These three variable will be used for all calendars and contacts in configuration file
+  
+        * ``USERNAME`` # Credential presented to the server  
+        * ``PASSWORD`` # Credential presented to the server
+        * ``CRON_FREQUENCY`` # See `this <http://www.nncron.ru/help/EN/working/cron-format.htm>`_ tutorial
+    
+    For CardDav (contacts) the following variables are needed
 
-3) Move the file to your Ubuntu Touch device, either by file manager or with adb:: 
+        * ``CARD_URL``  # The base url shared between all contact groups belonging to ``USERNAME``, If there is only one contact group for the user this can be the full URL
+        * ``CARD_NAMES`` # An array of contact group names (``CARD_URL`` and ``CARD_NAMES[n]`` will be concatinated), This can empty or unset if ``CARD_URL`` is the full URL (as is the case for Nextcloud users)
 
-    adb push dav.sh /home/phablet
+          The following three variables are used by *syncevolution* and they are restricted to 31 characters and can NOT include the '@' symbol. They must be unique across all of *syncevolution* else *syncevolution* will fail with strange errors.
+    
+        * ``CONTACTS_SERVER_CONFIG_NAMES`` # An array used to identify the configurations for each remote server URL
+        * ``CONTACTS_NAMES`` # An array used to identify the configurations for each local database kept in sync with each remote server URL  
+        * ``CONTACTS_VISUAL_NAMES`` # An array holding the display names that will be visible to applications using contact databases, meant for humans to read
+    
+    For CalDav (calendars) the following variables are needed
+  
+        * ``CAL_URL`` # The base url shared between all calendars belonging to ``USERNAME`` on a server
+        * ``CAL_NAMES`` # An array of calendar names (``CAL_URL`` and ``CAL_NAMES[n]`` will be concatinated) Currently by default Nextcloud has two calendars 'personal' and 'contact_birthdays'
 
-4) Connect with the phablet shell (``adb shell``) or directly on the phone Terminal app and type the following::
+          The following three variables are used by *syncevolution* and they are restricted to 31 characters and can NOT include the '@' symbol. They must be unique across all of *syncevolution* else *syncevolution* will fail with strange errors.
+ 
+        * ``CALENDAR_SERVER_CONFIG_NAMES`` # An array used to identify the configurations for each remote server URL
+        * ``CALENDAR_NAMES`` # An array used to identify the configurations for each local database kept in sync with each remote server URL
+        * ``CALENDAR_VISUAL_NAMES`` # An array holding the display names that will be visible to applications using calendar databases, meant for humans to read
 
-    chmod +x dav.sh
-    ./dav.sh
+5) Process all configurations
+    ``./setup-dav-sync.sh --contacts --calendar config-personal.txt config-2.txt config-3.txt``
```

Comment:
Since you haven't introduced using more than one config file, you might not want to mention it here.